x-pack/plugins/monitoring/public/components/summary_status/_summary_status.scss

```diff
@@ -1,5 +1,13 @@
-.monSummaryStatus {
-  background-color: $euiColorLightestShade;
-  border-bottom: $euiBorderThin;
-  padding: $euiSize;
-}
+.monSummaryStatusNoWrap {
+  margin-left: 12px;
+  margin-right: 12px;
```

Comment:
These two 12px settings should be replaced with EUI size variables. You can find all the available size variables here https://github.com/elastic/eui/blob/master/src/global_styling/variables/_size.scss , but for 12px you would use `$euiSizeM`